yarn.lock

```diff
@@ -4,26 +4,26 @@
 
 "@babel/code-frame@7.12.11":
   version "7.12.11"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.12.11.tgz#f4ad435aa263db935b8f10f2c552d23fb716a63f"
+  resolved "https://nexus.soma.salesforce.com/nexus/content/groups/npm-all/@babel/code-frame/-/code-frame-7.12.11.tgz#f4ad435aa263db935b8f10f2c552d23fb716a63f"
```

Comment:
this doesn't seem correct.

Reply:
My mistake. I always forget to switch my `npmrc` back. 🤦